src/classes/ADDR_AdminAcc_TEST.cls

```diff
@@ -0,0 +1,891 @@
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+ 
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2016
+* @group Addresses
+* @group-content ../../ApexDocContent/Addresses.htm
+* @description Tests specific to Administrative Account Address Management. 
+*/
+
+@isTest
+public with sharing class ADDR_AdminAcc_TEST {
+    
+    private static void configSettings() {
+        Hierarchy_Settings__c hs = new Hierarchy_Settings__c(
+            Accounts_Addresses_Enabled__c = UTIL_Describe.getAdminAccRecTypeID() + ';',
+            Simple_Address_Change_Treated_as_Update__c = true);
+        UTIL_CustomSettingsFacade.getSettingsForTests(hs);
+    }
+        
+    /*********************************************************************************************************
+    @description Update existing default Addresses.  
+    verify: account billing address fields updated
+    **********************************************************************************************************/            
+    @isTest 
+    public static void updateDefaultAddr() {
+        configSettings();
+        
+        // this creates a default Address for each Account
+        UTIL_UnitTestData_TEST.AccsWithAddrs accsAddrs = UTIL_UnitTestData_TEST.createTestAccsAddrs(2, UTIL_Describe.getAdminAccRecTypeID());
+        List<Address__c> addrs = accsAddrs.addrs;
+        
+        // now let's update the Addresses
+        for (Integer i = 0; i < addrs.size(); i++) {
+            Address__c addr = addrs[i];
+            addr.MailingStreet__c = 'New Street' + i;
+            addr.MailingCity__c = 'New City' + i;
+        }
+        system.assertEquals(false, ADDR_Addresses_TDTM.hasRunAddrTrigger);
+        Test.startTest();
+        update addrs;
+        Test.stopTest();
+    
+        map<Id, Account> mapAccIdAcc = new map<Id, Account>([select Id, Name, BillingStreet, BillingCity, BillingState, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. I suggest to introduce a `WHERE` clause or cap the max number of records returned adding a `LIMIT` clause to the query.